libs/ngx-dsfr/alert/src/lib/testing/alert.harness.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Angular imports
+ */
+import {
+  AsyncFactoryFn,
+  ComponentHarness,
+  TestElement
+} from '@angular/cdk/testing';
+
+/**
+ * This class inherits from ComponentHarness and its purpose
+ * is to serve as a wrapper around an instance of DsfrAlertComponent
+ * within Unit Tests
+ *
+ * To see more : https://material.angular.io/cdk/test-harnesses/overview
+ */
+export class DsfrAlertHarness extends ComponentHarness {
+  static hostSelector: string = 'dsfr-alert';
+
+  private getAlertTitleTag: AsyncFactoryFn<TestElement> = this.locatorFor('h3');
+  private getAlertDescriptionTag: AsyncFactoryFn<TestElement> = this.locatorFor('p');
```

Comment:
Since they both can be not present, I suggest you use `locatorForOptional`, instead of `locatorFor`.
See https://material.angular.io/cdk/test-harnesses/overview#finding-elements-in-the-components-dom

That way, you won't need a `try...catch` in your methods.
Don't forget to adapt the comments describing the methods now that they can both be optional 😊